source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,89 @@
 .. _minio-restore-hardware-failure-site:
 
-==========================
-Recover after Site Failure
-==========================
+=====================
+Site Failure Recovery
+=====================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-ToDo- site replication recovery procedure
+MinIO can make the loss of an entire site, while significant, a relatively minor incident.
+Site recovery depends on the replication option you use for the site.
+
+.. list-table:: 
+   :widths: 25 75
+   :width: 100%
+
+   * - Site Replication
+     - Total restoration from the healthy peer site(s)
+   * - Bucket Replication
+     - Data restoration of objects and metadata from a healthy remote location for each bucket configured for replication
+   * - :mc-cmd:`mc mirror`
+     - Data restoration of objects only (no metadata) from a healthy remote location
+
+Site Replication
+----------------
+
+:ref:`Site replication <minio-site-replication-overview>` keeps two or more MinIO deployments in sync with IAM policies, buckets, bucket configurations, objects, and object metadata.
+If a peer site fails, such as due to a major disaster or long power outage, you can use the remaining healthy site(s) restore the :ref:`replicable data <minio-site-replication-what-replicates>`.
+
+1. Deploy a new MinIO site using the same ``root`` credentials as used on other deployments in the site replication configuration
+
+   You can use the original hardware, if still available, but you must first wipe any remaining data before creating the new site.
+2. Configure the new site with the same Identity Provider (IDp) as the other sites
+3. :ref:`Expand the existing site replication <minio-expand-site-replication>` by adding the newly deployed site
+4. Remove the lost site from the site replication configuration
+
+Site replication healing automatically adds the buckets and objects from the existing site(s) to the new site with no further action.
+
```

Comment:
Above section is correct if site replication is in place prior to the disaster. However, there are situations where bucket replication may have been in place but customer wants to switch to site replication after disaster to take advantage of the automatic healing in site replication.

If previously bucket replication was in place on the sites, and after the disaster site replication is being brought in to recover the unhealthy site, all previous bucket replication configuration needs to be removed first on the healthy site - otherwise references to stale targets in the replication config can interfere with site replication.

Reply:
Sounds like we might need a dedicated page for migrating from bucket to site replication.

For here would it be enough (I think, if I'm reading this correctly) to note that:

- You cannot configure site replication for recovery if any bucket replication rules remain in place

- Bucket replication is mutually exclusive with site replication

Reply:
I'm adding a new section on the site replication page about switching from bucket replication.